sklearn/ensemble/forest.py

```diff
@@ -807,11 +807,22 @@ class RandomForestClassifier(ForestClassifier):
         Best nodes are defined as relative reduction in impurity.
         If None then unlimited number of leaf nodes.
 
-    min_impurity_split : float, optional (default=1e-7)
-        Threshold for early stopping in tree growth. A node will split
-        if its impurity is above the threshold, otherwise it is a leaf.
+    min_impurity_decrease : float, optional (default=0.)
+        A node will be split if this split induces a decrease of the impurity
+        greater than or equal to this value.
 
-        .. versionadded:: 0.18
+        The weighted impurity decrease equation is the following:
```

Comment:
@raghavrv will the math display correctly from lines 815-816? The `` tag will work properly, but does indenting alone work as intended?